source/read-write-concern/read-write-concern.md

```diff
@@ -523,11 +510,10 @@ Below are English descriptions of other items that should be tested:
 ### WriteConcern
 
 1. Commands supporting a write concern MUST NOT send the default write concern to the server.
-2. Commands supporting a write concern MUST send any non-default acknowledged write concern to the server, either in the
-    command or as a getLastError.
-3. On ServerVersion less than 2.6, drivers MUST NOT send a getLastError command for an Unacknowledged write concern.
-4. FindAndModify helper methods MUST NOT send a write concern when the MaxWireVersion is less than 4.
-5. Helper methods for other commands that write MUST NOT send a write concern when the MaxWireVersion is less than 5.
+2. Commands supporting a write concern MUST send any non-default acknowledged write concern to the server in the
+    command.
+3. FindAndModify helper methods MUST NOT send a write concern when the MaxWireVersion is less than 4.
```

Comment:
We can remove this since all drivers have dropped support for wire version <=4. 7 is the current min.